cfgov/unprocessed/apps/owning-a-home/js/explore-rates/dom-values.js

```diff
@@ -0,0 +1,38 @@
+const unFormatUSD = require( '../../node_modules/unformat-usd' );
+
+/**
+ * Get value(s) of an individual HTML element in the control panel.
+ * @param   {string} param Name of parameter to get.
+ *                         Usually the HTML element's id attribute.
+ * @returns {Object} Hash of element id and its value(s).
+ */
+function getSelection( param ) {
```

Comment:
This feels like the wrong name, it's not just getting a selection, it's returning the value of that selection.

Reply:
This is the original name, but it could be updated, what do you think?